core/pom.xml

```diff
@@ -230,6 +230,15 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-text</artifactId>
         </dependency>
+
+        <!-- Optional used in com.opensymphony.xwork2.util.ProxyUtil to detect if object is HibernateProxy -->
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>6.4.4.Final</version>
```

Comment:
As far I know this version won't work on JDK8 which is a base version for Struts 6.x

Reply:
You are correct - downgraded to Hibernate 5 as this proxy detection code is identical :)